1_OCaml/TP/TP4.ipynb

```diff
@@ -28,6 +28,52 @@
     "2. Réécrire `List.mem` (en appelant votre fonction `mem`, par exemple).\n"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    let m = [4 ; -12 ; 3] in somme m
</pre><p>Au lieu de </p><pre class="ql-syntax" spellcheck="false">let m = [4 ; -12 ; 3] in somme m
</pre><p>Tu peux écrire :</p><pre class="ql-syntax" spellcheck="false">somme [4 ; -12 ; 3]
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/104/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>